connectors/webhook/element-templates/webhook-connector-start-event.json

```diff
@@ -140,6 +148,76 @@
         "equals": "enabled"
       }
     },
+    {
+      "id": "authorizationType",
+      "label": "Authorization type",
+      "group": "authorization",
+      "description": "Choose the authorization type.",
+      "value": "none",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "None",
+          "value": "NONE"
+        },
+        {
+          "name": "JWT",
+          "value": "JWT"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.authorizationType"
+      }
+    },
+    {
+      "label": "JWK url",
+      "description": "Well-known url of JWKs",
+      "type": "String",
+      "group": "authorization",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.jwkUrl"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "JWT"
+      }
+    },
+    {
+      "label": "JWT role property expression",
+      "description": "Expression to extract the roles from the JWT token",
+      "type": "String",
+      "group": "authorization",
+      "feel": "required",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.jwtRoleExpression"
+      },
+      "condition": {
+        "property": "authorizationType",
+        "equals": "JWT"
+      }
+    },
+    {
+      "label": "Required roles",
+      "description": "List of roles to test JWT roles against",
+      "type": "String",
+      "group": "authorization",
+      "feel": "required",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.requiredPermissions"
```

Comment:
Can we move the JWT related properties to `inbound.jwt` leading to`inbound.jwt.requiredPermissions`.

Reply:
I would recommend to use the `authorizationType` as a discriminator for a generalized `Auhtorization` sealed class similar to: https://github.com/camunda/connectors-bundle/blob/2b6dcce6252fdb5eccd20bb7fb21704647a634dd/connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/model/AwsInput.java